accord-core/src/test/java/accord/impl/basic/PendingQueue.java

```diff
@@ -23,9 +23,11 @@
 public interface PendingQueue
 {
     void add(Pending item);
+    void addNoDelay(Pending item);
     void add(Pending item, long delay, TimeUnit units);
     boolean remove(Pending item);
     Pending poll();
     int size();
+    void checkFailures();
```

Comment:
This method only makes sense for the `PropagatingPendingQueue`. Does it really need to be exposed?

Reply:
we use `PendingQueue` and not `PropagatingPendingQueue`, so by changing the signature we use I can move it there; this is leaking details of one impl.

Reply:
removed from this interface and pass a new `Runnable checkFailures` to `Cluster` which calls this function.